go/vt/mysqlctl/schema.go

```diff
@@ -46,6 +51,26 @@ func (mysqld *Mysqld) executeSchemaCommands(sql string) error {
 	return mysqld.executeMysqlScript(params, strings.NewReader(sql))
 }
 
+func encodeTableName(tableName string) string {
+	var buf strings.Builder
+	sqltypes.NewVarChar(tableName).EncodeSQL(&buf)
+	return buf.String()
+}
+
+// tableListSql returns an IN clause "('t1', 't2'...) for a list of tables."
+func tableListSql(tables []string) (string, error) {
+	if len(tables) == 0 {
+		return "", errors.New("no tables for tableListSql")
+	}
+
+	encodedTables := make([]string, len(tables))
+	for i, tableName := range tables {
+		encodedTables[i] = encodeTableName(tableName)
+	}
+
+	return "(" + strings.Join(encodedTables, ", ") + ")", nil
+}
+
```

Comment:
I'm just wondering if we don't already have that kind of plumbing somewhere in our parser/sql code? If not, we must. Quoting a name, creating a quoted `IN` query are things we should not have to write all over our code.